org.eclipse.winery.topologymodeler.ui/src/app/navbar/navbar.component.ts

```diff
@@ -9,40 +9,36 @@
  * Contributors:
  *     Josip Ledic - initial API and implementation
```

Comment:
If you've done some changes, you should insert your name and what you've done here.
This also goes for all files.